annotations/src/main/java/org/corfudb/annotations/InterfaceOverride.java

```diff
@@ -1,19 +1,17 @@
-package org.corfudb.runtime.object;
+package org.corfudb.annotations;
 
 import java.lang.annotation.ElementType;
 import java.lang.annotation.Inherited;
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
-/**
- * Created by mwei on 1/7/16.
+/** Marks that a interface method (marked default) should
```

Comment:
It does what the comment says - when used, it causes a default method in an interface to override any implementation in the target class. It's currently not used.